bin/console

```diff
@@ -1,32 +1,33 @@
 #!/usr/bin/env php
 <?php
 
+// if you don't want to setup permissions the proper way, just uncomment the following PHP line
+// read http://symfony.com/doc/current/book/installation.html#configuration-and-setup for more information
+//umask(0000);
+
+set_time_limit(0);
```

Comment:
Moved this console from `tests/app/console` to its current location `bin/console`.